openms-wiki/coding-conventions.md

```diff
@@ -0,0 +1,174 @@
+Coding Conventions
+==================
```

Comment:
let's delete this file from this pr, as this should be added in `OpenMS/OpenMS-docs` `doxygen` docs.

Reply:
if you have worked on content improvement, please save it somewhere or just update the wiki.

Reply:
done